Admin/Admin.php

```diff
@@ -1620,20 +1653,6 @@ public function setFormGroups(array $formGroups)
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function removeFieldFromFormGroup($key)
```

Comment:
Because it was only used in the ``FormMapper::remove()`` method (see https://github.com/sonata-project/SonataAdminBundle/pull/2814/files#diff-90fea2ed781a41c220e9674b87a94c81L170) and is not called anymore, it is replaced with the new ``Admin::cleanupFormGroupsAndTabs`` method (see https://github.com/sonata-project/SonataAdminBundle/pull/2814/files#diff-81cd2da3c7ffebe278779869361e685aR1339)